tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaAuthorizationTestBase.java

```diff
@@ -336,8 +347,7 @@ public void testListTopic() throws Exception {
         ListTopicsResult listTopicsResult = adminClient.listTopics();
         Set<String> topics = listTopicsResult.names().get();
 
-        assertEquals(topics.size(), 1);
-        assertTrue(topics.contains(newTopic));
+        assertEquals(topics.size(), 0);
```

Comment:
Could you explain the behavior change here?

Reply:
The previous behavior is one by one to check topic permission, it can list the topic level topic, but in pulsar, it is only able to list by namespace, if we don't have the namespace permission, even if we authorized a topic level permission, it still not able to see it.